db/migrate/20150116161044_create_firms_service_regions.rb

```diff
@@ -0,0 +1,8 @@
+class CreateFirmsServiceRegions < ActiveRecord::Migration
+  def change
+    create_table :firms_service_regions, id: false do |t|
+      t.belongs_to :firm, index: true
```

Comment:
We should add a compound index on both the fields here too. You can see examples of that in the migrations for `Adviser` reference data.